.clang-tidy

```diff
@@ -82,10 +82,10 @@ Checks: >
 # openMP                                                doesn't apply (we're not using OpenMP)
 # zircon                                                utility checks that would need configuration
 
-## Aliasses
```

Comment:
These changes are all made automatically. I will comment on the things I changed manually here.